aiida/common/pluginloader.py

```diff
@@ -77,7 +77,7 @@ def get_class_typestring(type_string):
 
 
 def _existing_plugins_with_module(base_class, plugins_module_path,
-                                  pkgname, basename, max_depth, suffix=None):
+                                  pkgname, basename, max_depth, suffix=None, verbose=False):
```

Comment:
In the end we didn't implement verbose, could you remove this (also from the other function below)